src/parse/asp/config.go

```diff
@@ -108,10 +108,19 @@ func resolvePluginValue(values []string, subrepo string) []string {
 				continue // I guess it wasn't a build label. Leave it alone.
 			}
 			// Force the full build label including empty subrepo so this is portable
-			v = fmt.Sprintf("///%v//%v:%v", l.Subrepo, l.PackageName, l.Name)
+			var buf strings.Builder
```

Comment:
Or even just using string concatenation:
```
v = `///`+l.Subrepo+`//`+l.PackageName+`:`+l.Name
```
But this won't call into the `Stringer` interface if any of these variables aren't a `string`.

Reply:
It's called a lot more often than you'd expect. The way subrepos + plugins work with go_repo each being their own subrepo, it's called often.

> I'd expect under most circumstances the vast majority of wall-clock time to be spent on the build steps themselves, rather than anything to do with parsing BUILD files?

Well it depends what you're doing. Things like `plz query changes` are usually blocked by parsing files.